platforms/jvm/plugins-application/build.gradle.kts

```diff
@@ -24,12 +24,12 @@ description = "Contains the Application plugin, and its supporting classes.  Thi
 dependencies {
     api(projects.core)
     api(projects.coreApi)
+    api(projects.pluginsJavaBase)
+    api(projects.stdlibJavaExtensions)
```

Comment:
I don't know why I needed to make this change because I didn't touch anything in this project. Might be related to https://github.com/gradle/gradle-private/issues/4444 ?

Reply:
Yes, it was that same issue.